include/librpmi.h

```diff
@@ -378,6 +378,7 @@ void rpmi_shmem_destroy(struct rpmi_shmem *shmem);
  * a built-in service group of the RPMI context and is always available.
  */
 struct rpmi_context;
+struct rpmi_transport;
```

Comment:
Why this change ?

Reply:
Due to movement of the groups in header, this structure is gone below where it was referred. So instead of moving that group again above i declared here. Not sure why it didnt failed when i build with previous PR but it failed now so made the change.

Reply:
If this is to fix compile warning then move LIBRPMI_TRANSPORT_INTERFACE  before LIBRPMI_CONTEXT_INTERFACE

Reply:
Either this or moving the group again to satisfy the symbol visibility is fine.

Reply:
ok